src/kitaspectwidget.cpp

```diff
@@ -62,12 +63,8 @@ void KitAspectWidget::makeReadOnly()
     m_chooser->setReadOnly(true);
 }
 
-QWidget *KitAspectWidget::mainWidget() const
-{
-    return m_chooser->lineEdit();
-}
-
-QWidget *KitAspectWidget::buttonWidget() const
+void KitAspectWidget::addToLayout(Utils::LayoutBuilder &builder)
 {
-    return m_chooser->buttonAtIndex(0);
+    addMutableAction(m_chooser);
+    builder.addItem(Utils::Layouting::Span(2, m_chooser));
```

Comment:
why 2?

Reply:
because thanks to "2" span value asn1scc path field and path browser button are alignmented to others

Reply:
but can this magic number be inferred from something? or is it also 2 in the parent widget?